benchmark_runner/common/template_operations/render_yaml_from_template.py

```diff
@@ -20,8 +20,9 @@ def render_yaml_string(template_str: str, environment_variable_dict: dict = None
         environment_variable_dict['prom_token'] = environment_variable_dict.get('prom_token_override', '')
     else:
         oc = OC(kubeadmin_password=environment_variable_dict.get('kubeadmin_password', ''))
-        # set prom token
-        environment_variable_dict['prom_token'] = oc.get_prom_token()
+        # set prom token, skip if running on kubernetes just for now
+        if environment_variable_dict.get("cluster") != 'kubernetes':
```

Comment:
@gqlo, Why did you ask for negative, why not positive:
if environment_variable_dict.get("cluster") == 'openshift':

Reply:
The reason is that I wanna make it clear we are doing a dirty hack for kubernetes, if we are using "openshift" then by reading this line, we won't be able to understand this line is changed for the purpose of skipping prom token in kubernetes.

Reply:
I think that asking for positive is more clear and more readability

Reply:
`if environment_variable_dict.get("cluster") == 'openshift':`  By reading this line, as you can see nothing tells us about kubernetes. let's say if we have a new teammate come to our team, it would be very difficult for him to guess this line has something to do with k8s. I tend to stick to this principle as much as possible so that new teammates needs less context to understand the code. It probably won't make much difference for now, since we might need to remove the whole condition in the future if we want to run workload with prometheus in k8s.

I would be interested to know why you think a positive check is more clear and readable. Can you elaborate with more details?

Reply:
For me its more clear, we can leave it as is.